### ISSUE TITLE
Naming inconsistency between AutoValidateAntiforgeryTokenAttribute and ValidateAntiForgeryTokenAttribute

### ISSUE THREAD

**Issue by hikalkan** [aspnet/Mvc#5178]
`AutoValidateAntiforgeryTokenAttribute` uses "f" but `ValidateAntiForgeryTokenAttribute` uses "F" as first char of the Forgery word.

Just wanted to inform you.

Edit: It's similar for `IgnoreAntiforgeryTokenAttribute`

**Comment by Eilon** [comment]
Not sure how I missed seeing this bug for so long! This was actually deliberate because `ValidateAntiForgeryTokenAttribute` is a "legacy" name from MVC 5, which we wanted to preserve, and was used very often. In other places where we either have brand new type names, or perhaps existing, but less-commonly used type names, we changed to use what we believe is the more correct casing, which is lowercase `f`.

So, yeah, it's kind of weird, but we did it on purpose. Thanks for the report!
